validator/accounts/account.go

```diff
@@ -225,17 +225,26 @@ func HandleEmptyKeystoreFlags(cliCtx *cli.Context, confirmPassword bool) (string
 func Merge(ctx context.Context, sourceDirectories []string, targetDirectory string) (err error) {
 	var sourceStores []*db.Store
 	defer func() {
+		errorMessage := "failed to close one or more source databases"
```

Comment:
can you make this error a package variable ? 
ex:
```go
errSource := errors.New("failed to close one or more source databases")
```

You can then resuse the error here and in other places instead of redefining it again.